### PR TITLE
feat(website): add tutorial document

### LINKED ISSUE #393: Documentation for newcomer
## Motivation

As a newcomer to CKB, I'd like to get a primer on CKB and Lumos, it would be helpful to have a document that tells me how to use Lumos to interact with CKB

## How

It would be very helpful to have a tutorial, which could be structured like

- Creating a wallet
- Claiming the test CKB from the test network faucet
- Checking the CKB balance
- Sending my first transaction, which could be a simple CKB transfer (in fact, for the UTxO model, the transfer is not as intuitive as for the account model, so it is not quite as simple)
- Creating a token (similar to ERC20)
- Transferring token to Alice
- Checking Alice's token balance

### PR DESCRIPTION
**Pull request by PainterPuppets**
# Description

 - Add a step-by-step tutorial to help newcomers understand how to use lomus.

## Type of change

- [x] Documentation or Website

# How Has This Been Tested?

 - [x] Follow the document step-by-step to make a wallet

# Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules -->

### CODE REVIEW

File: website/docs/00-intro.md [hunk 1]
```diff
@@ -3,12 +3,636 @@ slug: /
 sidebar_position: 1
 ---
 
-# Tutorial
+# Tutorial: Step-by-step create a wallet with Lumos
```
Comment (homura) [inline website/docs/00-intro.md]:
```suggestion
# Tutorial
```

File: website/docs/00-intro.md [hunk 2]
```diff
@@ -3,12 +3,636 @@ slug: /
...
 
-# Tutorial
+# Tutorial: Step-by-step create a wallet with Lumos 
 
-Lumos is a collection of libraries that allow developers to build CKB-based dApps.
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-## Quick Start
+We will create a simple wallet to help manage assets on the ckb chain. We won't delve into the shape of the wallet product in this tutorial, but we have implemented some simple logic that most wallets have
```
Comment (homura) [inline website/docs/00-intro.md]:
```suggestion
We will create a simple HD wallet to help manage assets on CKB. We won't delve into the shape of the wallet product in this tutorial, but we have implemented some simple logic that most wallets have
```

File: website/docs/00-intro.md [hunk 3]
```diff
@@ -3,12 +3,636 @@ slug: /
...
+
+``` ts
+import { hd } from '@ckb-lumos/lumos';
+const { mnemonic, ExtendedPrivateKey } = hd;
+
+export const generatePrivateKey = () => {
+  const m = mnemonic.generateMnemonic()
+  const seed = mnemonic.mnemonicToSeedSync(m)
+  return ExtendedPrivateKey.fromSeed(seed).privateKey
+}
```
Comment (homura) [inline website/docs/00-intro.md]:
```suggestion
export const generateFirstHDPrivateKey = () => {
  const m = mnemonic.generateMnemonic()
  const seed = mnemonic.mnemonicToSeedSync(m)
  console.log('my mnemonic ', seed);
  
  
  const extendedPrivKey = ExtendedPrivateKey.fromSeed(seed);
  return extendedPrivKey.privateKeyInfo(
    AddressType.Receving,
    0,
  ).privateKey
}
```

the `ExtendedPrivateKey` is a master key to derive child key

File: website/docs/00-intro.md [hunk 4]
```diff
@@ -3,12 +3,636 @@ slug: /
...
+```
+
+
+CKB has the testnet `Arggon` and the mainnet  `Lina`, `Lumos` can work on different environment.
```
Comment (homura) [inline website/docs/00-intro.md]:
```suggestion
CKB has the testnet `Arggon` and the mainnet `Lina`, `Lumos` can work on different environment.
```

**Review by homura — APPROVED** [review]

File: website/docs/00-intro.md [hunk 5]
```diff
@@ -5,10 +5,646 @@ sidebar_position: 1
 
 # Tutorial
 
-Lumos is a collection of libraries that allow developers to build CKB-based dApps.
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-## Quick Start
+In this tutorial, we will develop a simple HD wallet to help manage assets on CKB. and we need to implement some logic that most wallets have
```
Comment (IronLu233) [inline website/docs/00-intro.md]:
```suggestion
In this tutorial, we will develop a simple HD wallet to help manage assets on CKB. And we need to implement some logic that most wallets have.
```

File: website/docs/00-intro.md [hunk 6]
```diff
@@ -5,10 +5,646 @@ sidebar_position: 1
...
 
+ - Generate private key and address
+ - Check CKB balance
+ - Transfer CKB to others
```
Comment (IronLu233) [inline website/docs/00-intro.md]:
```suggestion
 - Transfer CKB to other address
```

File: website/docs/00-intro.md [hunk 7]
```diff
@@ -5,10 +5,646 @@ sidebar_position: 1
...
+:::
+
+```mdx-code-block
+<Tabs>
+  <TabItem value="npm" label="npm" default>
+```
+
+``` shell
+npm init
+```
```
Comment (IronLu233) [inline website/docs/00-intro.md:46]:
```suggestion
``` shell
npm init
```
`-y` can skip input something for package info

image

File: website/docs/00-intro.md [hunk 8]
```diff
@@ -5,10 +5,646 @@ sidebar_position: 1
...
+
+```mdx-code-block
+  </TabItem>
+  <TabItem value="yarn" label="yarn">
+```
+
+``` shell
+yarn init
```
Comment (IronLu233) [inline website/docs/00-intro.md:54]:
```suggestion
yarn init -y
```

File: website/docs/00-intro.md [hunk 9]
```diff
@@ -5,10 +5,646 @@ sidebar_position: 1
...
+</Tabs>
+```
+
+### 2. Generate a wallet account
+
+Run the following command to create a new file：
+
+``` shell
+touch wallet.ts
+```
```
Comment (IronLu233) [inline website/docs/00-intro.md]: We can not preset the user use Linux / MacOS system. `touch` is not availiable on Micro$oft Windows.

Just tell user, open the folder with your preferred editor, create a file named `wallet.ts` in the directory root.

File: website/docs/00-intro.md [hunk 10]
```diff
@@ -5,10 +5,646 @@ sidebar_position: 1
...
+  let capacities = BI.from(0);
+  for await (const cell of collector.collect()) {
+    capacities = capacities.add(cell.cellOutput.capacity);
+  }
+
+  return capacities;
+}
+```
+
+Congratulations! we write a function to get the balance of account, now we can call it to get our balances
```
Comment (IronLu233) [inline website/docs/00-intro.md:200]: Is it missing a step to run
``` bash
npx ts-node wallet.ts
```
?

File: website/docs/00-intro.md [hunk 11]
```diff
@@ -5,10 +5,646 @@ sidebar_position: 1
...
+
+:::note
+The unit of capacity in CKB is Shannon, 1CKB = 10^8 Shannon, so here we have to divide by 10^8 to get the unit of CKB
+:::
+
+``` ts
+console.log(`address: ${address}`)
```
Comment (IronLu233) [inline website/docs/00-intro.md:208]: Unclear. As a noob, I want to ask, where should I put these two line code? Maybe I think I need to write it at the end of previous `wallet.ts` file

**Review by IronLu233 — COMMENTED** [review]
I think "tell developer where the code in document should be placed" is very important. Should we explictly tell developer that "write these code to which file"

File: website/docs/00-intro.md (hunk 7 above)
Comment (PainterPuppets) [reply website/docs/00-intro.md:46]: User may want to know what this parameter does if some parameters are added , but this is not really our business
The `npm init` is a nice experience for the user to create a project through some questions, we don't need to introduce this part of the complexity, if the user knows what they are doing then just go for it

File: website/docs/00-intro.md [hunk 12]
```diff
@@ -5,10 +5,647 @@ sidebar_position: 1
...
+export const generateFirstHDPrivateKey = () => {
+  const m = mnemonic.generateMnemonic()
+  const seed = mnemonic.mnemonicToSeedSync(m)
+  console.log('my mnemonic ', seed);
+
+  const extendedPrivKey = ExtendedPrivateKey.fromSeed(seed);
+  return extendedPrivKey.privateKeyInfo(
+    AddressType.Receving,
+    0,
+  ).privateKey
```
Comment (homura) [inline website/docs/00-intro.md]:
```suggestion
  const myMnemonic = mnemonic.generateMnemonic();
  const seed = mnemonic.mnemonicToSeedSync(myMnemonic);
  console.log("my mnemonic ", seed);

  const extendedPrivKey = ExtendedPrivateKey.fromSeed(seed);
  return extendedPrivKey.privateKeyInfo(AddressType.Receiving, 0).privateKey;
```

**Pull request merged**